test/IntercomCustomersTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Intercom\Test;
+
+use Intercom\IntercomUsers;
+use PHPUnit\Framework\TestCase;
+
+class IntercomCustomersTest extends TestCase
+{
+    public function testUserSearch()
+    {
+        $stub = $this->getMockBuilder('Intercom\IntercomClient')->disableOriginalConstructor()->getMock();
+        $stub->method('post')->willReturn('foo');
+
+        $users = new IntercomCustomers($stub);
+        $this->assertEquals('foo', $users->search({"query": {}}));
```

Comment:
This should do what you want:

```suggestion
        $this->assertEquals('foo', $users->search(["query": []]));
```